testsuite/standalone/src/test/java/org/wildfly/extras/creaper/commands/infinispan/cache/AddDistributedCacheOnlineTest.java

```diff
@@ -17,6 +17,7 @@
 
 import java.io.IOException;
 import java.util.UUID;
+import java.util.concurrent.TimeoutException;
```

Comment:
Oh and I wonder, how come this is now necessary? The `apply` method signature didn't change, right?